data/jedi.js

```diff
@@ -0,0 +1,60 @@
+import tempJediCard from '../public/images/jedi/allyship-space-temp.png';
+
+const allyshipSpaces = [
```

Comment:
Is the plan for now to just update this page when new allyship spaces are planned?

Looking to the future, how will this data file integrate with our events page? Would we import the `allyshipSpaces` array and append it to the events data we have stored? If so, the overall data structure would have to be the same. I like the base that you've laid down here so maybe we can base the events page data structure off of this.

Reply:
Yeah, I think eventually we'd hit the API and then filter out only the allyship spaces (i.e. we'd tag them with some special key-value pair). In the meantime, it'll probably be part of the typical workflow we use for updating the general "events page" on the ACM website.

Reply:
sgtm

Reply:
I suppose this does raise a small issue while the API is still under development.

We should be featuring the allyship spaces in the our events pages in addition the jedi page. So for now, will need to remember to import this array to the data file for the events page